contributing-guides/style-guide.md

```diff
@@ -104,6 +104,11 @@ Example:
 ```
 - Pre-translated alias page templates can be found [here](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/alias-pages.md).
 
+## Description syntax
+
+- Use double quotes for quoting strings in descriptions.
```

Comment:
I think I understand what you mean now. So like quoting an excerpt from documentation?

If that's not what you mean, could you share an example, sorry. ^-^'

Then I can respond to this properly.